packages/brick_core/lib/src/query/where.dart

```diff
@@ -191,6 +191,10 @@ class Where extends WhereCondition {
   Where isNot(dynamic value) =>
       Where(evaluatedField, value: value, compare: Compare.notEqual, isRequired: isRequired);
 
+  /// Convenience function to create a [Where] with [Compare.inList].
+  Where isIn(List<dynamic> values) =>
+      Where(evaluatedField, value: values, compare: Compare.inList, isRequired: isRequired);
```

Comment:
I know this is a nit, but I'd like to keep this as generic as possible. Implementations could support a `Set`, a `List`, or their super class, `Iterable`. 

```suggestion
      Where(evaluatedField, value: values, compare: Compare.inIterable, isRequired: isRequired);
```